.gitignore

```diff
@@ -7,6 +7,7 @@ output/*
 outputs/*
 data/*
 data_rollout/*
+test.ipynb
```

Comment:
Im guessing this is for dev? rmb to remove. might confuse someone in the future

Reply:
Or rather do 
```
*.ipynb
```